main.tf

```diff
@@ -527,7 +522,7 @@ resource "aws_codedeploy_deployment_group" "deploymentgroup" {
   deployment_config_name = "CodeDeployDefault.ECSAllAtOnce"
 
   ecs_service {
-    cluster_name = local.cluster_name
+    cluster_name = var.app_name
```

Comment:
ok, so we might need the cluster name in the existing_ecs_cluster variable so that we can use it here (and in the other places where cluster name is used)

Reply:
Is that the variable in the variables.tf file or the use of the existing_ecs_cluster variable in the example file (now simple.tf)? Regarding the comment below, would that mean we need both a cluster_name and name? I think I'm a bit confused on this one.